api-specs/api_spec_PGS.yaml

```diff
@@ -485,96 +583,19 @@ paths:
             type: string
       responses:
         "302":
-          description: OK-FOUND
-        "401":
-          description: Unauthorized
-          content:
-            application/json:
+          description: Resume completed
+          headers:
+            location:
```

Comment:
```suggestion
            Location:
```

Reply:
why? Field names are case-insensitive

Reply:
This is the header key. Since it's case-insensitive it's reported in CamelCase in any documentation, also in API guidelines